EstateManagement/Startup.cs

```diff
@@ -203,9 +220,31 @@ public void ConfigureServices(IServiceCollection services)
             });
             services.AddSingleton<HttpClient>();
         }
-
+        
+        /// <summary>
+        /// Configures the middleware services.
+        /// </summary>
+        /// <param name="services">The services.</param>
         private void ConfigureMiddlewareServices(IServiceCollection services)
         {
+            services.AddHealthChecks()
+                    .AddSqlServer(connectionString:ConfigurationReader.GetConnectionString("HealthCheck"),
+                                  healthQuery:"SELECT 1;",
+                                  name:"Read Model Server",
+                                  failureStatus:HealthStatus.Degraded,
+                                  tags:new string[] {"db", "sql", "sqlserver"})
+                    .AddEventStore(Startup.EventStoreClientSettings,
+                                   userCredentials: Startup.EventStoreClientSettings.DefaultCredentials,
+                                   name: "Eventstore",
+                                     failureStatus: HealthStatus.Unhealthy,
+                                     tags: new string[] { "db", "eventstore" })
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/stuart_ferguson1/EstateManagement/pullRequest?prid=6232863)